apps/ai-dial-admin/src/components/Common/FolderList/FolderList.tsx

```diff
@@ -205,10 +205,15 @@ const FolderList: FC<Props> = ({
           {isFolder(nodeType) && (
             <div className="flex flex-col">
               <div
-                className={`group ${baseClass} ${selectedClass} ${isMovableFolder && 'pointer-events-none'} ${isMoveError ? 'bg-error border-l-error' : ''} py-2`}
+                className={classNames(
+                  'group',
+                  baseClassName,
+                  selectedClassName,
+                  isMovableFolder && 'pointer-events-none',
```

Comment:
Do you need `isMoveError`?
```suggestion
                  isMovableFolder && 'pointer-events-none',
                  isMoveError && 'bg-error border-l-error'
```

Reply:
yes, line 213